src/charm.py

```diff
@@ -10,13 +10,15 @@
 from ops.charm import CharmBase
 from ops.framework import StoredState
 from ops.main import main
+from ops import ActiveStatus
 
 logger = logging.getLogger(__name__)
 
 CREDENTIALS_FILE = "/srv/influx.conf"
 DRY_RUN_FILE = "/srv/dry-run.conf"
```

Comment:
I don't if/where they are used, but maybe those paths should be updated to something like `/home/ubuntu/` to be consistent with the cloning location.

Reply:
Great idea! I've gone ahead and done so.